api/src/controllers/extension-permissions.ts

```diff
@@ -0,0 +1,222 @@
+import { isDirectusError } from '@directus/errors';
+import express from 'express';
+import { ErrorCode, RouteNotFoundError } from '../errors/index.js';
+import { respond } from '../middleware/respond.js';
+import { validateBatch } from '../middleware/validate-batch.js';
+import { ExtensionPermissionsService } from '../extensions/service.js';
+import { MetaService } from '../services/meta.js';
+import type { PrimaryKey } from '../types/index.js';
+import asyncHandler from '../utils/async-handler.js';
+import { sanitizeQuery } from '../utils/sanitize-query.js';
+import useCollection from '../middleware/use-collection.js';
+import { getExtensionManager } from '../extensions/extensions.js';
+
+const router = express.Router();
+
+router.use(useCollection('directus_extension_permissions'));
```

Comment:
Any chance we can combine this with `directus_extensions`?

Reply:
I don't know if this is possible, but it'd be nice if `permissions` was a column of `directus_extensions`

Reply:
Would be nice!
Also if we aren't able to combine it, IMO `directus_extensions_permission` would look a tad cleaner. (I know very nitpicking 🤓)

Reply:
Resolved in https://github.com/directus/directus/pull/19507/commits/f7e361e238300b1018e56f72a2de0529c2babef6